engine/src/cmd/drawable.cpp

```diff
@@ -99,13 +99,12 @@ void Drawable::AnimationStep()
 #endif
         if((!this->isContinuousLoop())&&(loopCount==0))
             return;
-
-        int numvold = 0;
-        int numvertices = 0;
         //copy reference to data
         meshdata.at(0) = vecAnimations.at(activeAnimation)->at(activeMesh);
 
 #ifdef DEBUG_MESH_ANI
+        int numvold = 0;
+        int numvertices = 0;        
         std::cerr << "vertices changed from: " << numvold << " to: " << numvertices << std::endl;
```

Comment:
does this actually do anything? If it's just printout out `0` and `0` then why not remove it?

Reply:
I've chopped out this ifdef, as requested